core/templates/pages/exploration-player-page/templates/information-card-modal.component.html

```diff
@@ -103,7 +103,7 @@ <h2 [ngStyle]="titleWrapper()" class="oppia-info-card-title">{{ explorationTitle
   }
   .oppia-info-card-content .oppia-contributors-icon {
     cursor: default;
-    margin-right: 0.3125rem;
+    margin-right: 9px;
```

Comment:
Prefer using `rem` or `em` instead of px (see https://github.com/oppia/oppia/issues/10754).